res/layout/webxdc_view.xml

```diff
@@ -12,12 +12,14 @@
                       android:gravity="center_vertical"
                       android:orientation="vertical">
 
-            <androidx.appcompat.widget.AppCompatImageView
+            <com.google.android.material.imageview.ShapeableImageView
                 android:id="@+id/webxdc_icon"
+                android:padding="0.1dp"
```

Comment:
I don't know if I can add `<!-- -->` comments here, would be nice to tell that this is a hack needed to avoid the transparent background if padding is zero, for some weird reason I guess a bug in material design lib